{"uuid":"477b1bf0-6f9b-48f1-b9a3-64156cfba9c1","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["fc68d09f-ccf2-4d03-b3ed-212d6a71aa50"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696715059789,"stop":1696715059789},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696715059912,"stop":1696715059912},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696715059974,"stop":1696715059975},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696715062005,"stop":1696715062005}],"start":1696715059661,"stop":1696715062006}